torch/ao/quantization/pt2e/quantizer/x86_inductor_quantizer.py

```diff
@@ -245,16 +323,38 @@ def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
     def _annotate_for_static_quantization_config(
         self, model: torch.fx.GraphModule
     ) -> torch.fx.GraphModule:
+        config = self.global_config
+
+        # Step1: Recipe of fusion patterns
         # annotate the nodes from last to first since the matching is in the reversed order
         # and fusion operator patterns (conv - relu) can get matched before single operator pattern (conv)
-        # and we will mark the matched node with "_annoated" so fusion operator pattern
+        # and we will mark the matched node with "_annotated" so fusion operator pattern
         # can take precedence over single operator pattern in this way
-        config = self.global_config
+        self._annotate_conv2d_fusion_pattern(model, config)
+
+        for node in model.graph.nodes:
+            # Step2: Recipe of quantizable single op.
+            # Go through all the nodes from first to last.
+            # Recipe refer to https://github.com/intel/intel-extension-for-pytorch/blob/
+            # 90d19323d96afc53fcc22ba5a7bb3fb07fdd6c1c/intel_extension_for_pytorch/quantization/_recipe.py#L538
+            self._annotate_quantizable_single_op(node, config)
+
+        for node in model.graph.nodes:
+            # Step3: For quantizable ops, such as maxpool2d, we need to quantize its output if it is quantized
+            # in inputs. So, we can fuse dq-operator-q into a quantized op.
+            # Refer to https://github.com/intel/intel-extension-for-pytorch/blob/
+            # 90d19323d96afc53fcc22ba5a7bb3fb07fdd6c1c/intel_extension_for_pytorch/quantization/_recipe.py#L487
+            self._annotate_quantized_node_output(node, config)
```

Comment:
It actually handles `int8_in_int8_out_ops_pt2e` right? If so, why not name it that way?

Reply:
Thanks for the commend and changed.